src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -517,26 +536,26 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
             borderBottomRadius='xl'
           >
             <ButtonWalletPredicate
-              isValidWallet={Boolean(isChainSupportedByWallet)}
+              isValidWallet={Boolean(isChainSupportedByWallet && !isAccountsLoading)}
               isDisabled={Boolean(
                 errors.amountFieldInput ||
                   !runeAddress ||
                   !isValidStakingAmount ||
                   !(isStakeFeesSuccess || isGetApprovalFeesSuccess) ||
+                  isAccountsLoading ||
                   !cooldownPeriod,
               )}
               size='lg'
               mx={-2}
               onClick={handleWarning}
               isLoading={isGetApprovalFeesLoading || isStakeFeesLoading}
               colorScheme={
-                Boolean(errors.amountFieldInput || errors.manualRuneAddress) ? 'red' : 'blue'
+                Boolean(errors.amountFieldInput || errors.manualRuneAddress) && !isAccountsLoading
```

Comment:
This is different from the blue colorScheme in swapper @NeOMakinG we may want to revert this bit

Reply:
While this does technically fix the issue, it does feel weird having the account metadata loading state being tied to the validity of a wallet. Not going to block on it and not a change for this PR, but maybe something to think about regarding use/props for `ButtonWalletPredicate`.